tests/urlpatterns_reverse/tests.py

```diff
@@ -225,6 +229,18 @@ def test_patterns_reported(self):
             # exception
             self.fail("Expected a NoReverseMatch, but none occurred.")
 
+    def test_script_name_escaping(self):
+        from django.core.urlresolvers import get_script_prefix, set_script_prefix
+        old_prefix = get_script_prefix()
```

Comment:
Here's a ticket if you have interest in trying to do the implementation: https://code.djangoproject.com/ticket/24476